### DISCUSSION TITLE
Multiple interface mixing net_offload and Zephyr IP stack based devices

### DISCUSSION THREAD

**Discussion by RomainPelletant** [zephyrproject-rtos/zephyr#51523] [Q&A] [answered]
Hi,

In a project, I have three interfaces IP based:

- Ethernet (zephyr IP stack driver)
- Wifi (offload driver)
- LTE-M (PPP based on zephyr IP stack)

Can I use ethernet and LTE-M if the NET_OFFLOAD parameter is enabled ?
If no, do you know a workaround?

Thanks

**Comment by jukkar** [comment, marked as answer]
Yes, that is supported. If you find problems with this combo, just file an issue.

**Reply by RomainPelletant** [reply]
Thanks!
